analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/InvalidCastToInterface.CSharp9.cs

```diff
@@ -48,7 +48,7 @@ public void TargetTypedNew()
         var i = (int)nullable;
 
         nullable = null;
-        i = (int)nullable; // FN, can't build CFG for this method
+        i = (int)nullable; // Noncompliant
```

Comment:
I can't leave a comment there 🤷 but the field `private string field;` in line 37 seems unused.